### DISCUSSION TITLE
Microbial community assembly

### DISCUSSION THREAD

**Discussion by makerer5** [ChiLiubio/microeco#302] [Ideas]
Dear Chi,

I hope this message finds you well. As a newcomer to the world of R programming, especially in the realm of data visualization, I've often found myself searching for plotting codes on Google. It was during one of these searches that I stumbled upon "microeco" on GitHub, and I must say, I am thoroughly impressed by its capabilities.

I regret not discovering "microeco" sooner, as it would have spared me much time and effort. The tutorials you've posted have been incredibly helpful; I've watched them more than ten times! They are straightforward, well-explained, and highly informative. Please accept my sincere gratitude for creating and sharing such a valuable resource.

While I find "microeco" to be an exceptional tool, I believe there is an opportunity for further enhancement. Here are two suggestions for future development:

Microbial Community Assembly: This topic is gaining significant attention in microbial ecology. If you could provide a detailed tutorial on "iCAMP" within "microeco", it would be a tremendous resource for the community. A tutorial like this could aid many researchers and students in their work, and I'm confident it would be greatly appreciated.

Neutral Community Model: This is a crucial aspect of community assembly mechanisms. Currently, the resources available on Google are somewhat limited and unsatisfactory. A tutorial addressing this topic within "microeco" would not only fill an important gap but also enhance the understanding of community dynamics.

I believe these additions would greatly enrich the "microeco" package, making it even more beneficial for users like myself who rely on such tools for their research and learning.

Thank you once again for your fantastic work. I eagerly look forward to any future updates and tutorials you might provide. Your contribution to the field is invaluable.

Best regards,

Mark

**Comment by ChiLiubio** [comment]
Dear Mark,

Thanks very much for your appreciation and suggestions. I totally agree with your thought. Currently, the "trans_nullmodel" part in the tutorial does indeed have some shortcomings. We will consider how to enhance the two aspects you mentioned. Please feel free to tell us if you have any other idea/source/suggestions.

Best regards,
Chi
